packages/valory/skills/abstract_round_abci/test_tools/abci_app.py

```diff
@@ -31,119 +31,71 @@
 )
 
 
-class ConcreteRoundA(AbstractRound):
-    """Dummy instantiation of the AbstractRound class."""
-
-    round_id = "concrete_a"
-    allowed_tx_type = "payload_a"
+class _ConcreteRound(AbstractRound):
+    """ConcreteRound"""
 
-    def end_block(self) -> Tuple[MagicMock, MagicMock]:
+    def end_block(self) -> None:
         """End block."""
-        return MagicMock(), MagicMock()
 
     def check_payload(self, payload: BaseTxPayload) -> None:
-        """Check payloads of type 'payload_a'."""
+        """Check payload."""
 
     def process_payload(self, payload: BaseTxPayload) -> None:
-        """Process payloads of type 'payload_a'."""
+        """Process payload."""
 
 
-class ConcreteRoundB(AbstractRound):
+class ConcreteRoundA(_ConcreteRound):
     """Dummy instantiation of the AbstractRound class."""
 
-    round_id = "concrete_b"
-    allowed_tx_type = "payload_b"
+    round_id = "concrete_a"
+    allowed_tx_type = "payload_a"
 
-    def end_block(self) -> None:
+    def end_block(self) -> Tuple[MagicMock, MagicMock]:  # type: ignore
```

Comment:
you wouldn't have to do a type ignore if your parent class was properly defining its types